compiler_opt/rl/inlining/gin_configs/ppo_nn_agent.gin

```diff
@@ -20,18 +20,6 @@ train_eval.deploy_policy_name='saved_collect_policy'
 train_eval.use_random_network_distillation=False
 train_eval.moving_average_decay_rate=0.8
 
-# TODO(b/233935329): The following clang flags need to be tied to a corpus
-# rather than to a training tool invocation.
-
-# List of flags to add to clang compilation command. The flag names should
-# match the actual flags provided to clang. An example for AFDO reinjection:
-# train_eval.flags_to_add=('-fprofile-sample-use=/path/to/gwp.afdo','-fprofile-remapping-file=/path/to/prof_remap.txt')
-train_eval.additional_compilation_flags=()
-
-# List of flags to remove from clang compilation command. The flag names 
-# should match the actual flags provided to clang.'
-train_eval.delete_compilation_flags=('-split-dwarf-file','-split-dwarf-output','-fthinlto-index','-fprofile-sample-use','-fprofile-remapping-file')
```

Comment:
Removing these lines will break the google internal trainer (train_with_rpc.py) since this gin file is shared by train_locally.py and train_with_rpc.py. 

I think to avoid such unexpected conflicts we should have a separate internal-only copy of gin files. What do you think @yundiqian?

Reply:
we don't need to, we can also do similar things internally, but yes, there needs to be a fix

Reply:
agreed